views/elements/course-filters.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * Course Filter views
+ *
+ * A common filter element for all the backend pages
+ *
+ * @package Tutor\Views
+ * @subpackage Tutor\ViewElements
+ * @author Themeum <support@themeum.com>
+ * @link https://themeum.com
+ * @since 3.5.0
+ */
+
+use TUTOR\Input;
+
+if ( isset( $data ) ) : ?>
+<div class="tutor-admin-container tutor-admin-container-lg">
+	<div class="tutor-wp-dashboard-course-filter tutor-justify-<?php echo esc_attr( isset( $data['bulk_action'] ) && true === $data['bulk_action'] ? 'between' : 'end' ); ?>">
```

Comment:
Replace this
`<?php echo esc_attr( isset( $data['bulk_action'] ) && true === $data['bulk_action'] ? 'between' : 'end' ); ?>`
With this:
`<?php echo esc_attr( ! empty( $data['bulk_action'] ) ? 'between' : 'end' ); ?>`